### DISCUSSION TITLE
Filter column numerically

### DISCUSSION THREAD

**Discussion by pyeh-seg** [grafana/grafana#64261] [Dashboards, Panels & Visualizations]
Currently, columns with numerical data are filtered discretely. For example, if I have a column with data 0, 0.7, and 1, I can only filter the column by checking/unchecking 0, 0.7, or 1. As we are dealing with huge amount of data, I hope we could have a way to filter the column numerically. For example, filter to value >0.5, and the column would only display 0.7 and 1. I hope we could do this without having to check each of the values that we want to look at.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I'm having trouble understanding where this filtering is happening. We have a transformation "filter data by values" that allows for filtering greater than/less than/etc. I'm not sure if youre referring to filtering somewhere else or if that feature wasnt implemented in the version of Grafana you were using when you wrote this.

Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution still and would like to request the feature.
